pkg/tnf/test.go

```diff
@@ -33,6 +33,13 @@ const (
 	FAILURE
 )
 
+// EXITCODEMAP maps a test result value to a more appropriate Unix return code.
+var EXITCODEMAP = map[int]int{
+	SUCCESS: 0,
```

Comment:
For constants, the convention is not clear in Effective Go:  https://golang.org/doc/effective_go#constants.  Thus, let's just keep them as is for now.